Find a functionally equivalent code:com.android.tools.layoutlib.create.DependencyFinder.DependencyVisitor.MyMethodVisitor.visitFieldInsn:COMMENT
Method Modifier: public      
Comment:// field instruction

@@ -1,5 +1,6 @@
 {
-    // name is the field's name.
+    // owner is the class that declares the field.
+    considerName(owner);
     // desc is the field's descriptor (see Type).
     considerDesc(desc);
 }

